pkg/daemon/ceph/client/pool.go

```diff
@@ -209,7 +219,7 @@ func CreatePoolWithPGs(context *clusterd.Context, clusterInfo *ClusterInfo, clus
 		pool,
 		pgCount,
 		appName,
-		true /* enableECOverwrite */)
+		true /* enableECOverwrite */, autoscaleMode)
```

Comment:
Keep with established syntax and put this on a new line

```suggestion
		true /* enableECOverwrite */, 
		autoscaleMode)
```